doc/simastrom.tex

```diff
@@ -30,7 +30,7 @@
 \def\rm{\normalfont \rmfamily}
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-\title{Documentation of the meas/simastrom package}
+\title{Documentation of the jointcal package}
```

Comment:
Isn't this document describing the WCS part of jointcal? We should be explicit if we are assuming that `jointcal` is going to do photometry as well. Also, is that why this file was not renamed to `jointcal.tex`?